infrastructure.tf

```diff
@@ -426,171 +434,276 @@ resource "oci_core_network_security_group_security_rule" "cluster_lb_nsg_rule_5"
     }
   }
 }
-
+ 
 resource "oci_core_network_security_group_security_rule" "cluster_lb_nsg_rule_6" {
   network_security_group_id = oci_core_network_security_group.cluster_lb_nsg.id
   protocol                  = local.all_protocols
   direction                 = "INGRESS"
   source                    = var.vcn_cidr
 }
 
+resource "oci_core_network_security_group_security_rule" "cluster_lb_nsg_rule_7" {
+  network_security_group_id = oci_core_network_security_group.cluster_lb_nsg.id
+  protocol                  = "6"
+  direction                 = "INGRESS"
+  source                    = local.anywhere
```

Comment:
What about to limit this rule to VCN CIDR only? Same applies for 22623.

Reply:
In that case, can we remove the specific ingress rules for 22623 and 22624 and rely on nsg_rule_6 which allows ingress on all ports from within vcn_cidr?